docs/src/main/asciidoc/grpc-service-consumption.adoc

```diff
@@ -295,10 +295,19 @@ public class MyService {
 
 == Client Interceptors
 
-You can implement a gRPC client interceptor by implementing an `@ApplicationScoped` bean implementing `io.grpc.ClientInterceptor`:
+A gRPC client interceptor can be implemented by a CDI bean that also implements the `io.grpc.ClientInterceptor` interface.
+If you want to apply the interceptor to any injected client then annotate the interceptor bean with `@io.quarkus.grpc.GlobalInterceptor`.
+You can also register the interceptor per injected client with `@io.quarkus.grpc.RegisterClientInterceptor`.
```

Comment:
You should mention that this annotation is used on the client (on the injection site), while GlobalInterceptor is on the interceptor.